base_menu_visibility_restriction/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base Menu Visibility Restriction",
+    "version": "17.0.1.0.0",
+    "category": "Extra Tools",
+    "development_status": "Production/Stable",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "summary": "Restrict (with groups) menu visibilty",
+    "website": "https://github.com/OCA/server-ux",
+    "license": "AGPL-3",
+    "depends": ["web_tour"],
+    "maintainers": ["victoralmau"],
+    "data": ["views/ir_ui_menu.xml"],
+    "demo": ["demo/res_group.xml"],
```

Comment:
I think this file is not needed due to the elimination of the ir_ui_menu.xml file.

Now the new group does nothing

Reply:
@Tisho99 I removed it. Thanks!

Reply:
Thanks to you!